wordpress-mobile/WordPress-iOS
# Fix 'Take a Photo' option failing after adding an image to gallery block

Fixes https://github.com/wordpress-mobile/gutenberg-mobile/issues/1965

Could potentially fix https://sentry.io/organizations/a8c/issues/1535683817/?project=1438083

To test:

1. Add a gallery block
1. Add an image using "Choose from device"
1. Tap the "+" button at the bottom of the gallery block
1. Add an image using the "Take a Photo" option
1. Newly taken photo should be added to the gallery block
1. Now add an image block
1. Try adding an image to it using the "Take a Photo" option
1. Newly taken photo should be added to the image block

PR submission checklist:
